### ISSUE TITLE
validator rejects labels with more than one `/`

### ISSUE THREAD

**Issue by gunjan5** [projectcalico/libcalico-go#322]
This commit line: https://github.com/projectcalico/libcalico-go/commit/9930a7e60f8ffd7b74bc9ced86637a93450f9b3f#diff-fe058abafc6b5378c4afc418edd45ef3L37

It seems to break our stars demo at: http://docs.projectcalico.org/v2.0/getting-started/kubernetes/tutorials/stars-policy/

with the following felix errors: 
```bash
root@node-01# k logs calico-node-66z7x -c calico-node -n kube-system | grep validation
2017-01-11 22:06:28.658 [WARNING][195] validation_filter.go 53: Validation failed; treating as missing error=error with field src_selector = 'k8s_ns/label/role == 'management-ui'' key=Policy(name=stars.allow-ui) value=order:1000,selector:"calico/k8s_ns == 'stars'",inbound:allow from selector "k8s_ns/label/role == 'management-ui'",outbound:allow
2017-01-11 22:06:53.382 [WARNING][195] validation_filter.go 53: Validation failed; treating as missing error=error with field src_selector = 'k8s_ns/label/role == 'management-ui'' key=Policy(name=client.allow-ui) value=order:1000,selector:"calico/k8s_ns == 'client'",inbound:allow from selector "k8s_ns/label/role == 'management-ui'",outbound:allow
2017-01-11 22:16:04.476 [WARNING][195] validation_filter.go 53: Validation failed; treating as missing error=error with field src_selector = 'k8s_ns/label/role == 'client'' key=Policy(name=stars.frontend-policy) value=order:1000,selector:"calico/k8s_ns == 'stars' && role == 'frontend'",inbound:allow tcp from selector "k8s_ns/label/role == 'client'" to ports 80,outbound:allow
``` 

Need to:
- [x] Fix libcalico-go validator code to not reject multiple `/`s 
- [x] Do a felix release with that commit pinned 
- [x] Need to add this to Calico `v2.0.1` release

**Comment by caseydavenport** [comment]
This breaks NamespaceSelector in Kubernetes NetworkPolicy, so this is pretty serious.

**Comment by caseydavenport** [comment]
A potential workaround for users would be using `calicoctl` to interact with Calico policy objects directly (instead of using `kubectl` and `NetworkPolicy`), and to use labels without more than one `/`.

For example, modify the following label on your workload endpoints such that `k8s_ns/label/role: management-ui` becomes `k8s_ns.label/role: management-ui`

Then, update to reference that label in policies instead of `k8s_ns/label/role`.

**Comment by caseydavenport** [comment]
Note that this has been fixed in Calico v2.0.1 (`calico/node:v1.0.1`).

It's recommended users upgrade to the latest v2.0.x (or v2.1.x) release as described on docs.projectcalico.org.
